src/status_im/ui/components/list_item/views.cljs

```diff
@@ -1,82 +1,284 @@
+     (cond
+       (keyword? title-prefix)
+       [icons/icon title-prefix
+        (merge
+         {:color           colors/gray
+          :width           16
+          :height          16
+          :container-style (merge
+                            {:width            16
+                             :height           16
+                             :justify-content  :center
+                             :margin-top       (if (not title-prefix-height) 4 0)
+                             :align-items      :center}
+                            (when title-prefix-width
+                              {:width title-prefix-width})
+                            (when title-prefix-height
+                              {:height title-prefix-height}))}
+         (when title-prefix-width
+           {:width title-prefix-width})
+         (when title-prefix-height
+           {:height title-prefix-height}))]
+
+       (string? title-prefix)
+       [react/text {:number-of-lines 1
+                    :ellipsize-mode  :tail
+                    :style
+                    (merge (styles/title-prefix type icon? subtitle content)
+                           (when title-prefix-width
+                             {:width title-prefix-width})
+                           (if disabled?
+                             {:color colors/gray}
+                             (cond
+                               (= theme :action-destructive)
+                               {:color colors/red}
+                               (= theme :action)
+                               {:color colors/blue})))}
+        (if title-prefix-width
+          title-prefix
+          (str title-prefix " "))]
+
+       (vector? title-prefix)
+       title-prefix
+
+       :else
+       [title-prefix]))
+
+   (cond
+     (string? title)
+     [react/text {:number-of-lines 1
+                  :ellipsize-mode  :tail
+                  :style
+                  (merge (styles/title type icon? title-prefix subtitle
+                                       content title-row-accessory)
+                         (if (or disabled? (= :section-header type))
+                           {:color colors/gray}
+                           (if title-color-override
+                             {:color title-color-override}
+                             (cond
+                               (= theme :action-destructive)
+                               {:color colors/red}
+                               (= theme :action)
+                               {:color colors/blue}))))}
+      title]
+
+     (vector? title)
+     [react/view {:flex 1 :flex-grow 1 :flex-shrink 1}
+      title]
+
+     :else
+     [react/view {:flex 1 :flex-grow 1 :flex-shrink 1}
+      [title]])
+
+   (when title-row-accessory
+     [react/view {:margin-top       2
+                  :align-self       :stretch}
+      title-row-accessory])])
+
+(defview subtitle-row [{:keys [subtitle subtitle-max-lines subtitle-row-accessory]}
```

Comment:
you don't need a defview for that just use a defn

Reply:
understood 👍 . will update.

Reply:
done